### PR TITLE
Feat(): IA Mode

### PR DESCRIPTION
**Pull request by Enzo-py**

### CODE REVIEW

File: R5.08.Project.Forms/Bot.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
+﻿using ProjetForm;
+using System;
```
Comment (ElFloww) [inline R5.08.Project.Forms/Bot.cs:2]: You can remove these useless usings

File: R5.08.Project.Forms/Puissance4.cs [hunk 2]
```diff
@@ -26,6 +28,17 @@ public Puissance4(string p_Joueur1, string p_Joueur2, bool p_PlayerVSPlayerMode)
             this.v_Board = new bool[Puissance4Manager.BOARD_NUMBER_LINE, Puissance4Manager.BOARD_NUMBER_COLUMN];
             this.v_RedPawnOnBoard = new bool[Puissance4Manager.BOARD_NUMBER_LINE, Puissance4Manager.BOARD_NUMBER_COLUMN];
             this.v_YellowPawnOnBoard = new bool[Puissance4Manager.BOARD_NUMBER_LINE, Puissance4Manager.BOARD_NUMBER_COLUMN];
+            this.v_difficulty = v_difficulty;
+        }
+
+        public void Copy(Puissance4 other)
```
Comment (ElFloww) [inline R5.08.Project.Forms/Puissance4.cs:34]: `public void Copy(Puissance4 p_Other)`
Prefer to use a better name than other

**Review by ElFloww — COMMENTED** [review]
Review

**Comment by GaelGSST** [comment]
Care about variable and parameter names

**Review by GaelGSST — APPROVED** [review]

**Comment by ElFloww** [comment]
REJECTED

**Pull request closed without merging**